### PR TITLE
Add flake8 task

### PR DESCRIPTION
**Pull request by frerikandriessen**
It's a word-for-word copy of the pylint-0.2 task.

# Changes

Flake8 is another very popular python linting tool. There was no task for it yet so I figured I'd add it, based on the already existing pylint task. I made the OWNERS of this task the same as the OWNERS of the pylint task. Of course, I don't know if they're actually available for that, let me know if I need to change it. 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [X] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [X] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [X] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [X] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [X] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [X] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/flake8/0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# flake8
+
+The task provides linting based on [flake8](https://pypi.org/project/flake8/) for Python.
+This is a blatant copy of the [pylint](https://github.com/tektoncd/catalog/tree/main/task/pylint/0.2) task.
+
+## Install the Task
```
Comment (vinamra28) [inline task/flake8/0.1/README.md:6]: can this be removed :slightly_smiling_face: ?

File: task/flake8/0.1/flake8.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: flake8
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: python, flake8
+    tekton.dev/displayName: flake8
```
Comment (vinamra28) [inline task/flake8/0.1/flake8.yaml:11]: can we add one more annotation for it `tekton.dev/categories` ? Probably you can refer the PR #658

**Review by vinamra28 — COMMENTED** [review]
@frerikandriessen thanks for the PR :+1: 
just a couple of comments. Also feel free to add yourself in the OWNERS file if you want to author this task :slightly_smiling_face:

File: task/flake8/0.1/flake8.yaml (hunk 2 above)
Comment (frerikandriessen) [reply task/flake8/0.1/flake8.yaml]: Done

File: task/flake8/0.1/README.md (hunk 1 above)
Comment (frerikandriessen) [reply task/flake8/0.1/README.md:5]: Done

**Comment by frerikandriessen** [comment]
> @frerikandriessen thanks for the PR 👍
> just a couple of comments. Also feel free to add yourself in the OWNERS file if you want to author this task 🙂

I added myself to the OWNERS file but now it's complaining that I'm not a member of the org. What should/can I do about that?

**Comment by vinamra28** [comment]
@frerikandriessen  can you please add yourself to the org so that the OWNER file can be made valid? Reference PR -> tektoncd/community#472

**Comment by frerikandriessen** [comment]
Added myself to the org: https://github.com/tektoncd/community/pull/485

**Review by vdemeester — APPROVED** [review]

**Comment by vinamra28** [comment]
@frerikandriessen can you please squash the commits as well?

**Comment by frerikandriessen** [comment]
> The following users are mentioned in OWNERS file(s) but are not members of the tektoncd org.
> 
> * frerikandriessen
>   
>   * task/flake8/OWNERS
>   * task/flake8/OWNERS

I added myself to the org.yaml https://github.com/tektoncd/community/blob/main/org/org.yaml, was that not enough?

**Comment by frerikandriessen** [comment]
> @frerikandriessen can you please squash the commits as well?

Done

**Comment by vdemeester** [comment]
@frerikandriessen the org did not get synced, trying to get it fixed (https://github.com/tektoncd/community/pull/486 👼🏼 )

**Pull request merged**